src/Symfony/Component/Validator/Tests/ConstraintValidatorTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+final class ConstraintValidatorTest extends TestCase
+{
+    /**
+     * @dataProvider formatValueProvider
+     */
+    public function testFormatValue($expected, $value, $format = 0)
+    {
+        $this->assertSame($expected, (new TestFormatValueConstraintValidator())($value, $format));
+    }
+
+    public function formatValueProvider(): array
+    {
+        $data = [
+            ['true', true],
+            ['false', false],
+            ['null', null],
+            ['resource', fopen('php://memory', 'r')],
+            ['"foo"', 'foo'],
+            ['array', []],
+            ['object', $toString = new TestToStringObject()],
+            ['ccc', $toString, ConstraintValidator::OBJECT_TO_STRING],
+            ['object', $dateTime = (new \DateTimeImmutable('@0'))->setTimezone(new \DateTimeZone('UTC'))],
```

Comment:
How can I fix the 
```
-'Jan 1, 1970, 12:00 AM'
+'Jan 1, 1970, 1:00 AM'
```
failures in the CI?

Reply:
we should preserve the original timezone here https://github.com/symfony/symfony/pull/33434/files#diff-72fb9731ce9689c5dce965e9a97b2ee4R91

Reply:
Should be green after https://github.com/symfony/symfony/pull/33442.